### PR TITLE
Chams rewrite (and some other visuals stuff)

### PR DESCRIPTION
**Pull request by aUniqueUser**
Major changes
- Rewrote chams using DME hook only, allows for much more customization
- Users have complete control over the cham shader parameters, defaults look similar to some chams in CS:GO cheats

Misc changes
- Allow team colors to be changed
- Add more text locations to ESP
- Allow aimbot target color to be modified
- Allow gui color to be modified
- GUIColor() is now replaced with colors::gui
- Added a map texture override, not complete so no menu entries since im lazy 

TODO before a merge
- [x] Cleanup the arm and local weapon cham code
- [x] Comment my fucking code

### CODE REVIEW

File: src/visual/CMakeLists.txt [hunk 1]
```diff
@@ -2,8 +2,7 @@ set(files "${CMAKE_CURRENT_LIST_DIR}/atlas.cpp"
     "${CMAKE_CURRENT_LIST_DIR}/colors.cpp"
     "${CMAKE_CURRENT_LIST_DIR}/drawing.cpp"
     "${CMAKE_CURRENT_LIST_DIR}/drawmgr.cpp"
-    "${CMAKE_CURRENT_LIST_DIR}/EffectChams.cpp"
-    "${CMAKE_CURRENT_LIST_DIR}/EffectGlow.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/EffectGlow.cpp"
```
Comment (aUniqueUser) [inline src/visual/CMakeLists.txt:5]: Don't know what my IDE did here, will fix

**Comment by aUniqueUser** [comment]
This is pretty much ready for review @TotallyNotElite

**Comment by aUniqueUser** [comment]
Now really ready tm

**Comment by BenCat07** [comment]
this was merged

**Pull request closed without merging**